src/MatdaAIga.LinkConverter/Controller/IConverterController.cs

```diff
@@ -0,0 +1,11 @@
+/// <summary>
+/// Interface for a controller that handles conversion operations.
+/// </summary>
+public interface IConverterController
+{
+    /// <summary>
+    /// Invokes the conversion processs
+    /// </summary>    
+    /// <param name="args">The path of the file to convert</param>
```

Comment:
`/// <param name="args">The list of arguments passed from the command line</param>`